Backendless/RT/Data/DataSubscription.cs

```diff
@@ -5,7 +5,7 @@ public class DataSubscription : RTSubscription
   {
     public DataSubscription( RTDataEvents rtDataEvent, String tableName, IRTCallback callback ) : base( SubscriptionNames.OBJECTS_CHANGES, callback )
     {
-      PutOption( "event", Enum.GetName( typeof( RTDataEvents ), rtDataEvent ) );
+      PutOption( "event", Enum.GetName( typeof( RTDataEvents ), rtDataEvent ).Replace( "_", "-" ) );
```

Comment:
@holailusoria why is this needed? Seems odd

Reply:
@markpiller we get the name of the event handler of type Enum and convert it to a string. Requires a name of type "bulk-create", "bulk-update" etc. But we have "bulk_create", "bulk-update" respectively. This was the reason why BulkListeners did not work. This change fixes this bug.